.ci.yaml

```diff
@@ -36,6 +36,20 @@ targets:
           {"dependency": "vs_build"}
         ]
 
+  - name: Windows unit tests master - packages
+    recipe: packages/packages
+    timeout: 30
+    bringup: true
```

Comment:
```suggestion
```

See `bringup` in [the docs](https://github.com/flutter/cocoon/blob/main/CI_YAML.md):

```
bringup: Whether this target is under active development and should not block the tree.
         If true, will not run in presubmit and will not block postsubmit.
```

Reply:
the ci.yaml validation asks all newly added target to have bringup:true. If i removed this line, the presubmit will fail

Reply:
We're then in a catch-22 situation that I don't know how to get out of 😅

Reply:
Any new configuration has to start in bringup mode (so that the configuration has time to propagate IIRC), then have bringup removed once it's work. That's just how .ci.yaml works.